Core/Utilities/MySQL.py

```diff
@@ -1488,7 +1477,7 @@ def insertFields( self, tableName, inFields = None, inValues = None, conn = None
                          ( table, inFieldString, inValueString ), conn, debug = True )
 
 
-  def executeStoredProcedure( self, packageName, parameters, outputIds, output = True, array = None, conn = False ):
+  def executeStoredProcedure( self, packageName, parameters, outputIds ):
```

Comment:
I agree with you. We just need to be now concious that the interfaces are not compatible, which I think they were up to now, but I don't think we really care, especially with the SQLAlchemy direction